arangodb-net-standard.Test/TransactionApi/TransactionApiClientTest.cs

```diff
@@ -1,31 +1,336 @@
-﻿using ArangoDBNetStandard;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using ArangoDBNetStandard;
 using ArangoDBNetStandard.DocumentApi.Models;
 using ArangoDBNetStandard.TransactionApi.Models;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 using Xunit;
 
 namespace ArangoDBNetStandardTest.TransactionApi
 {
-    public class TransactionApiClientTest: IClassFixture<TransactionApiClientTestFixture>
+    /// <summary>
+    /// The Transaction Api Client test class.
+    /// </summary>
+    public class TransactionApiClientTest : IClassFixture<TransactionApiClientTestFixture>
     {
-        private ArangoDBClient _adb;
+        /// <summary>
+        /// The Arango database client.
+        /// </summary>
+        private readonly ArangoDBClient _adb;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TransactionApiClientTest"/> class.
+        /// </summary>
+        /// <param name="fixture">The test repository fixture with the blank database etc.</param>
         public TransactionApiClientTest(TransactionApiClientTestFixture fixture)
         {
             _adb = fixture.ArangoDBClient;
             _adb.Collection.TruncateCollectionAsync(fixture.TestCollection1).Wait();
             _adb.Collection.TruncateCollectionAsync(fixture.TestCollection2).Wait();
         }
 
+        /// <summary>
+        /// Tests that the stream transaction gets aborted successfully.
+        /// </summary>
+        [Fact]
+        [Trait("Feature", "StreamTransaction")]
+        public async Task AbortTransaction_ShouldSucceed()
+        {
+            // Begin a new transaction.
+            var beginTransaction = await _adb.Transaction.BeginTransaction(
+                new StreamTransactionBody
+                {
+                    Collections = new PostTransactionRequestCollections
+                    {
+                        Write = new[] { "TestCollection1" },
+                    },
+                });
+
+            // Abort the transaction.
+            var abortTransaction = await _adb.Transaction.AbortTransaction(beginTransaction.Result.Id);
+
+            // Check for the correct transaction status.
+            Assert.Equal(StreamTransactionStatus.Aborted, abortTransaction.Result.Status);
+        }
+
+        /// <summary>
+        /// Test that an exception is thrown when trying to abort a transaction that has already been committed.
+        /// </summary>
+        /// <exception cref="ApiErrorException">With ErrorNum 1653 if the transaction cannot be aborted.</exception>
+        [Fact]
+        [Trait("Feature", "StreamTransaction")]
+        public async Task AbortTransaction_ShouldThrowException_WhenTheTransactionCannotBeAborted()
```

Comment:
Same for other `ShouldThrowException` tests, I think we can keep only one per method and maybe rename it for clarity.

Reply:
I think the name of the test are okay. Since it explains what the exception will be about.
`AbortTransaction_ShouldThrowException_WhenTheTransactionIsNotFound`
`BeginTransaction_ShouldThrowException_WhenReferringToANonExistingCollection`
`CommitTransaction_ShouldThrowException_WhenTheTransactionIsNotFound`